src/compositemark/boxplot.ts

```diff
@@ -95,11 +96,11 @@ export function normalizeBoxPlot(spec: GenericUnitSpec<Encoding<string>, BOXPLOT
   // TODO: use selection
   const {mark, encoding, selection, ...outerSpec} = spec;
 
-  let kIQRScalar: number = undefined;
+  let kIQRScalar: number = (config.box && config.box.extent) ? config.box.extent : 1.5;
```

Comment:
if you already put 1.5 in the default config, you won't need redundancy here.  

The more important part is to make sure extent is a number.